### PR TITLE
Changelog updates for release v48.13.0

### PR DESCRIPTION
**Pull request by jag-sfdc**
### What does this PR do?
Changelog updated for release v48.13.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+# 48.13.0 - April 30, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-lwc
+
+- Add check for local dev server start failure ([PR #2128](https://github.com/forcedotcom/salesforcedx-vscode/pull/2128))
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]: We can remove this piece - this was internal as part of the LWR switch from Talon. It's not something customers would have experienced yet

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+
+#### docs
+
+- Beta for source retrieve on Apex, Visualforce, Aura and LWC ([PR #2102](https://github.com/forcedotcom/salesforcedx-vscode/pull/2102))
+
+#### salesforcedx-vscode-core
+
+- Beta for source retrieve on Apex, Visualforce, Aura and LWC ([PR #2102](https://github.com/forcedotcom/salesforcedx-vscode/pull/2102))
+
+- Add diagnostics for beta deploys ([PR #2143](https://github.com/forcedotcom/salesforcedx-vscode/pull/2143))
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:19]: We should add a link to this issue that got resolved by this fix: https://github.com/forcedotcom/salesforcedx-vscode/issues/2124

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+
+#### salesforcedx-vscode-lwc
+
+- Add check for local dev server start failure ([PR #2128](https://github.com/forcedotcom/salesforcedx-vscode/pull/2128))
+
+## Added
+
+#### docs
+
+- Beta for source retrieve on Apex, Visualforce, Aura and LWC ([PR #2102](https://github.com/forcedotcom/salesforcedx-vscode/pull/2102))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:13]: You can remove this and just add a link to the Perf Enhancement docs in the line for this work under the `salesforcedx-vscode-core` section

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 2 above)
Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md:19]: I would add that fix as a separate item and add the link Ananya mentioned.

Comment (jag-sfdc) [reply packages/salesforcedx-vscode/CHANGELOG.md:15]: Added that and moved to the fixed section. Let me know if that's fine.

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 3 above)
Comment (jag-sfdc) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Hope I got the link to public Perf Enhancement docs link right. Please confirm.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,19 @@
+# 48.13.0 - April 30, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Fix for the incorrect success message issue  ([Issue #2124](https://github.com/forcedotcom/salesforcedx-vscode/issues/2124))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]:
```suggestion
- Fix displaying a success message when a Beta Deploy contains failures  ([PR #2143](https://github.com/forcedotcom/salesforcedx-vscode/pull/2143), [Issue #2124](https://github.com/forcedotcom/salesforcedx-vscode/issues/2124))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,19 @@
...
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Beta for source retrieve on Apex, Visualforce, Aura and LWC ([Performance Enhancements](https://developer.salesforce.com/tools/vscode/en/user-guide/perf-enhancements))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:13]:
```suggestion
- Beta for source retrieve on Apex, Visualforce, Aura and LWC [Performance Enhancements](https://developer.salesforce.com/tools/vscode/en/user-guide/perf-enhancements) ([PR #2102](https://github.com/forcedotcom/salesforcedx-vscode/pull/2102))
```

**Pull request merged**